docs/dynamic-plugins.md

```diff
@@ -206,22 +294,37 @@ In order to add dynamic plugin support to a third-party front plugin, without to
     rootDirectory: dynamic-plugins-root
   ```
 
-- Copy the dynamic plugin package to the `dynamic-plugins-root` folder with the following commands:
+- Copy the dynamic plugin package to the `dynamic-plugins-root`. The following commands can be helpful:
 
-  ```bash
-  pkg=<local dist-dynamic sub-folder or external package name of the dynamic plugin package>
-  archive=$(npm pack $pkg)
-  tar -xzf "$archive" && rm "$archive"
-  mv package $(echo $archive | sed -e 's:\.tgz$::')
-  ```
+```bash
+pkg=<local dist-dynamic sub-folder or external package name of the dynamic plugin package>
+archive=$(npm pack --silent $pkg )
+tar -xzf "$archive" && rm "$archive"
+finalName=$(echo $archive | sed -e 's:\.tgz$::')
+rm -fr "$finalName"
+mv package "$finalName"
+```
 
-  It will create a sub-folder named after the package name, and containing the dynamic plugin package.
+It will create a sub-folder named after the package name, and containing the dynamic plugin package.
```

Comment:
To me this sentence reads a little awkwardly, how about: "It will create a sub-folder containing the dynamic plugin package that is named after the package name."